src/CoreBundle/Bridge/Chameleon/Module/Sidebar/SidebarBackendModule.php

```diff
@@ -0,0 +1,166 @@
+<?php
+
+/*
+ * This file is part of the Chameleon System (https://www.chameleonsystem.com).
+ *
+ * (c) ESONO AG (https://www.esono.de)
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace ChameleonSystem\CoreBundle\Bridge\Chameleon\Module\Sidebar;
+
+use ChameleonSystem\CoreBundle\Service\LanguageServiceInterface;
+
+class SidebarBackendModule extends \MTPkgViewRendererAbstractModuleMapper
+{
+    /**
+     * @var LanguageServiceInterface
+     */
+    private $languageService;
+
+    public function __construct(LanguageServiceInterface $languageService)
+    {
+        parent::__construct();
+        $this->languageService = $languageService;
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function Accept(\IMapperVisitorRestricted $oVisitor, $bCachingEnabled, \IMapperCacheTriggerRestricted $oCacheTriggerManager)
+    {
+        $oVisitor->SetMappedValue('menuItems', $this->getMenuItems());
+    }
+
+    private function getMenuItems(): array
+    {
+        $activeLanguageId = $this->languageService->getActiveLanguageId();
+        $activeUser = \TCMSUser::GetActiveUser();
+        if (null === $activeUser) {
+            return [];
+        }
+
+        $menuItemsRaw = [];
+        $this->addTableMenuItems($menuItemsRaw, $activeUser, $activeLanguageId);
+        $this->addModuleMenuItems($menuItemsRaw, $activeUser, $activeLanguageId);
+
+        return $this->mergeMenuItems($menuItemsRaw);
+    }
+
+    private function addTableMenuItems(array &$aMenuItemsTemp, \TCMSUser $activeUser, string $activeLanguageId): void
+    {
+        $query = "SELECT * FROM `cms_tbl_conf` WHERE `cms_content_box_id` <> '' AND `cms_content_box_id` <> '0'";
+        $tableList = \TdbCmsTblConfList::GetList($query, $activeLanguageId);
```

Comment:
We need to define an order for the elements using a position field (it is possible to configure the field with cms_contenten_box_id as filter). Could be added in a second PR for finetuning.

Reply:
See my notes in the PR description :-)

Reply:
Ah never mind...